backups/restore.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# Function to restore OpenMRS database
+function restore_openmrs_database() {
+  if [[ -d "$restore_folder/openmrs" ]]; then
+    echo "Restoring OpenMRS database"
+    docker exec -i "$openmrs_container" mysql -u $openmrsdb_username --password=$openmrsdb_password openmrs < "$openmrs_backup_file"
+  else
+    echo "Openmrs backup folder is missing. Thus, skipping OpenMRS restoration."
+  fi
+}
+
+# Function to restore Reports database
+function restore_reports_database() {
+  if [[ -d "$restore_folder/reports" ]]; then
+    echo "Restoring Reports database"
+    docker exec -i "$reports_container" mysql -u $reportsdb_username --password=$reportsdb_password bahmni_reports < "$reports_backup_file"
```

Comment:
Should we check for the file "reports_backup_file" too and display message accordingly.

Reply:
Considering folders will only be created if backup is taken so we only checked for folders and not the file.

Reply:
Okay thanks.